### ISSUE TITLE
The size of the terminal window increases when you click on the settings icon.

### ISSUE THREAD

**Issue by DmitrySkibitsky** [martyr-deepin/deepin-terminal-gtk#169]
The size of the terminal window increases when you click on the settings icon.
This error also occurs when calling the context menu in the terminal

![deepin-screen-recorder_Select area_20190602155444](https://user-images.githubusercontent.com/20288556/58761636-2cada980-854f-11e9-9313-d6c86ffa84b6.gif)

```log
yaourt deepin-terminal 
1 community/deepin-terminal 3.2.5-1 (deepin-extra) [installed]
```

**Comment by chirs241097** [comment]
Possible duplicate of #168.

**Comment by BLumia** [comment]
Should be resolved in 3.2.5.1, please update the package :)
